examples/agents/inflation.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# This software may be used and distributed in accordance with the terms of the Llama 3 Community License Agreement.
+
+import asyncio
+
+import fire
+from examples.custom_tools.ticker_data import TickerDataTool
+from llama_stack import LlamaStack
+
+from llama_stack.types import Attachment, SamplingParams, UserMessage
+from llama_stack.types.agent_create_params import *  # noqa: F403
+from common.client_utils import *  # noqa: F403
+from termcolor import cprint
+
+from .multi_turn import execute_turns, prompt_to_turn
+
+
+async def run_main(host: str, port: int, disable_safety: bool = False):
+    api_keys = load_api_keys_from_env()
+    tool_definitions = [
+        search_tool_defn(api_keys),
+        AgentConfigToolWolframAlphaToolDefinition(
```

Comment:
This seems inconsistent. 
better to have `search_tool_defn` and `wolfram_tool_defn` and `code_interpreter_defn` for all builtins.